workspaces/linux-host/Makefile

```diff
@@ -30,7 +29,7 @@ TEST_PARAMETERS = DATABASE_URL=$(PROXY_ATTESTATION_SERVER_DB) \
 	VERACRUZ_TRUST_DIR=$(OUT_DIR) \
 	VERACRUZ_PROGRAM_DIR=$(OUT_DIR) \
 	VERACRUZ_DATA_DIR=$(OUT_DIR) \
-	RUNTIME_MANAGER_ENCLAVE_PATH=$(RUNTIME_MANAGER_ENCLAVE)
+	RUNTIME_ENCLAVE_BINARY_PATH=$(WORKSPACE_DIR)/linux-runtime/runtime_manager_enclave
```

Comment:
With the changes in #339, the file `$(WORKSPACE_DIR)/linux-runtime/runtime_manager_enclave` will not be created anymore. This line should be:
```
	RUNTIME_ENCLAVE_BINARY_PATH=$(RUNTIME_MANAGER_ENCLAVE)
```

Reply:
Good catch. Looks like not having `runtime_manager_enclave` breaks the tests.
Let's keep copying it from `./target`.
Making changes to #339

Reply:
Fixed in #339